bin/newman.js

```diff
@@ -34,7 +34,7 @@ cli(process.argv.slice(2), 'newman', function (err, args) {
     dispatch(args, function (err) {
         if (err) {
             console.error(err);
-            process.exit(1);
+            process.exit(args.run.suppressExitCode ? 0 : 1);
```

Comment:
will this error out if `args.run` isn't present? In other words, is `args.run` guaranteed as object here (even for future where more commands will be in?